src/modules/navigator/mission.cpp

```diff
@@ -433,11 +408,11 @@ Mission::advance_mission()
 float
 Mission::get_absolute_altitude_for_item(struct mission_item_s &mission_item)
 {
-	if (_mission_item.altitude_is_relative) {
-		return _mission_item.altitude + _navigator->get_home_position()->alt;
+	if (mission_item.altitude_is_relative) {
```

Comment:
I checked the existing usage and flight tested it, although I doubt I hit all the cases.